ansible/roles/azure/templates/azureDeployLinuxCounters.json

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
I am also curious about how this file was generated.

Reply:
I'm pretty sure this was originally created from UI and then exported as a template. There is [public documentation](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/template-workspace-configuration) on the ARM template schema for Log Analytics workspaces but it is much easier to start with an exported template and then trim it down.

Reply:
Thanks for the information @arvindshmicrosoft